openshift_metrics/config.py

```diff
@@ -14,5 +14,26 @@
 S3_SECRET_ACCESS_KEY = os.getenv("S3_OUTPUT_SECRET_ACCESS_KEY")
 S3_INVOICE_BUCKET = os.getenv("S3_INVOICE_BUCKET", "nerc-invoicing")
 S3_METRICS_BUCKET = os.getenv("S3_METRICS_BUCKET", "openshift_metrics")
+
+# Billing Configuration
+# Service Unit Names for Rate Lookup from nerc-rates
+SU_NAMES = ["GPUV100", "GPUA100", "GPUA100SXM4", "GPUH100", "CPU"]
+RESOURCE_NAMES = ["vCPUs", "RAM", "GPUs"]
+
+# Rate Configuration
+USE_NERC_RATES_ENV = os.getenv("USE_NERC_RATES")
+USE_NERC_RATES = USE_NERC_RATES_ENV.lower() == "true" if USE_NERC_RATES_ENV else None
```

Comment:
I would suggest just having `USE_NERC_RATES` and removing `USE_NERC_RATES_ENV`. If the user does not set `USE_NERC_RATES`, that should be equivalent to setting the env var to false:
```suggestion
USE_NERC_RATES = os.getenv("USE_NERC_RATES", "false").lower() == "true"
```

Do you have a specific reason for why these two env vars needs to be defined seperately?